src/Service.Host/client/src/components/__tests__/parts/partReducer.test.js

```diff
@@ -73,7 +73,13 @@ describe('partReducer tests', () => {
 
     it('should handle simple field changes', () => {
         expect(
-            partReducer({}, { type: 'fieldChange', fieldName: 'partNumber', payload: 'PART' })
+            partReducer({}, { type: 'fieldChange', fieldName: 'partNumber', payload: ' PART' })
+        ).toEqual({ part: { partNumber: 'PART' } });
+    });
+
+    it('should trim leading/trailing spaces from partNumber', () => {
+        expect(
+            partReducer({}, { type: 'fieldChange', fieldName: 'partNumber', payload: ' PART' })
```

Comment:
maybe test ' PART ' just in case even though it doesn't make any difference?

Reply:
> do you think the server side should check this too just in case?

good call